main.js

```diff
@@ -125,26 +119,25 @@ export function assertStatusList2021Context({credential} = {}) {
 export function getCredentialStatus({credential, statusPurpose} = {}) {
   _isObject({credential});
   assertStatusList2021Context({credential});
-  // older revisions of `StatusList2021` used different types to express
-  // the status purpose:
-  const statusPurposes = {
-    revocation: 'RevocationList2021Status',
-    suspension: 'SuspensionList2021Status'
-  };
-  const expectedType = statusPurposes[statusPurpose];
+  if(!(statusPurpose && typeof statusPurpose === 'string')) {
+    throw new Error('"statusPurpose" string is required.');
+  }
   // get and validate status
   if(!(credential.credentialStatus &&
     typeof credential.credentialStatus === 'object')) {
     throw new Error('"credentialStatus" is missing or invalid.');
   }
   const credentialStatuses = _getStatuses({credential});
+  if(credentialStatuses.length === 0) {
+    throw new Error('"credentialStatus.type" must be "StatusList2021Entry".');
+  }
```

Comment:
```suggestion
```

Reply:
I think this error could be confusing when there are other valid status types ... so I suggest we combine its text with the error below.

Reply:
Updated [7458900](https://github.com/digitalbazaar/vc-status-list/pull/6/commits/745890086265f6f4e544e933399aba9dbe78e6cb)